microsoft-edge/devtools-guide-chromium/memory-problems/index.md

```diff
@@ -23,22 +23,46 @@ ms.date: 07/21/2023
    limitations under the License.  -->
 # Fix memory problems
 
-Learn how to use Microsoft Edge and DevTools to find memory issues that affect page performance, including memory leaks, memory bloat, and frequent garbage collections.
+To find memory issues that affect page performance, including memory leaks, memory bloat, and frequent garbage collections, use the following tools:
+* Microsoft Edge Browser Task Manager.
+* The **Performance** tool's **Memory** checkbox.
+* The **Memory** tool's various profiling types.
 
-*  Find out how much memory your page is currently using with the Microsoft Edge Browser Task Manager.
-*  Visualize memory usage over time with the **Memory** tool.
-*  Identify detached DOM trees (a common cause of memory leaks) with **Heap snapshot**.
-*  Find out when new memory is being allocated in your JavaScript heap (JS heap) with **Allocation instrumentation on timeline**.
 
-See also [Debug DOM memory leaks with the Detached Elements tool](dom-leaks.md).
+<!-- ------------------------------ -->
+#### Tools for investigating memory usage
```

Comment:
pls review & update the descriptions in the Task column; need summary recommendations of which tool to use when
@captainbrosset